### DISCUSSION TITLE
Bot is not used

### DISCUSSION THREAD

**Discussion by Torres90** [n3d1117/chatgpt-telegram-bot#375] [Q&A]
![Screenshot_2023-07-12-06-17-26-166_org telegram messenger](https://github.com/n3d1117/chatgpt-telegram-bot/assets/139307017/c3efc962-a160-4145-90e2-cce7d327197e)
Tell me what's the problem. Bot doesn't start. Standard commands are answered with the same message. There is no interaction with me.

**Comment by Vysokostnyi** [comment]
.env file most likely you don't have
